components/telegram_bot_api/telegram_bot_api.app.js

```diff
@@ -58,6 +74,161 @@ module.exports = {
         "Only emit events for the selected update types.",
       options: updateTypes,
     },
+    chatId: {
+      type: "string",
+      label: "Chat ID",
+      description: "Enter the unique identifier for the target chat or username of the target channel (in the format `@channelusername` or `@supergroupusername`).",
```

Comment:
I had a hard time trying to find a Chat ID in this format and I ended up setting a normal id like `1035597319` to test an action component and it worked. So I guess it would be nice to have more context of how the user con get this data or at least to have a link where the user can get a real Chat ID

Reply:
I updated the `description` of `chatId`:
https://github.com/PipedreamHQ/pipedream/blob/c80b5afcf9851debaa7cab8efab46a3619aeaa1f/components/telegram_bot_api/telegram_bot_api.app.mjs#L32-L35